packages/typespec-test/test/NetworkAnalytics.Management/tspconfig.yaml

```diff
@@ -1,6 +1,8 @@
 emit: ["@azure-tools/typespec-ts"]
 options:
   "@azure-tools/typespec-ts":
+    typespecTitleMap: 
+        NetworkAnalyticsClient: NetworkAnalyticsManagementClient
```

Comment:
@joheredi do you have any concerns about this kind of mapping that we provide to customers to modify the client name ? Which we have to in my point of view, because in the case of multiple clients, we can't use a single string option to know which client name we are trying to override.